src/components/image-selection/ImageSelectionButton.js

```diff
@@ -77,7 +78,7 @@ const ImageSelectionButton = ({
           <ReactDropzone
             onDrop={onSelect}
             className={styles.dropzone}
-            accept='image/*'
+            accept={ALLOWED_IMAGE_FILE_TYPES.join(', ')}
```

Comment:
you *could* export ALLOWED_IMAGE_FILE_TYPES.join(', ') from the imageProcessingUtil and that might be less error prone since this requires a comma and a space?

Reply:
I like keeping it in array form and formatting where necessary - easier to understand why it's formatted that way